src/components/Dashboard/Dashboard.jsx

```diff
@@ -3,68 +3,120 @@ import { CardColumns, Button } from 'react-bootstrap';
 import styled from 'styled-components';
 import AnnouncementCard from '../AnnouncementCard/AnnouncementCard';
 import UpcomingShiftCard from '../UpcomingShiftCard/UpcomingShiftCard';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Do we need this? I thought we resolved this issue

Reply:
i've removed it, when i started working on the task the bootstrap styling was not showing up unless i added it. working fine now.